maro/rl/policy/abs_policy.py

```diff
@@ -96,9 +96,17 @@ def train(self) -> None:
 
 
 class RuleBasedPolicy(AbsPolicy, metaclass=ABCMeta):
-    """Rule-based policy. The user should define the rule of this policy, and a rule-based policy is not trainable."""
+    """Rule-based policy. The user should define the rule of this policy, and a rule-based policy is not trainable.
 
-    def __init__(self, name: str) -> None:
+    Args:
+        name(str): name of this policy
+        action_by_idx (bool, default=True):
+            When set to true, the action quantity will be set as action index,
+            or the action will be determined by quantity.
+    """
+
+    def __init__(self, name: str, action_by_idx: bool = True) -> None:
```

Comment:
Do not add the arg for this base policy.
Implement it inside `examples/supply_chain`